src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -151,4 +154,16 @@ public class Utils {
             return false
         }
     }
+
+    @Whitelisted
+    static void setSkippedStage(String stageName) {
+        CpsThread thread = CpsThread.current()
+        CpsFlowExecution execution = thread.execution
+
+        FlowNode currentNode = execution.currentHeads.find { n ->
+            n?.displayName?.equals(stageName)
```

Comment:
Not sure if there's a better/more secure way to be sure we've got the right `FlowNode`...